module_4/src/db/index.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+import fs from 'fs';
+import Sequelize from 'sequelize';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import generalConfig from '../../config/dbConfig.json';
+
+class DB {
+  constructor() {
+    this.__filename = fileURLToPath(import.meta.url);
+    this.config = generalConfig[process.env.NODE_ENV || 'development'];
```

Comment:
Please, note that it is good to have only two NODE_ENV: `development` - for development purposes only and `production` - for all deployed envs.
Usually additional variable is added, e.g.: `NODE_CONFIGURATION` - to manage different configurations depending on environment you are going to deploy your app. 
When NODE_ENV === 'prodiction' many libs activate different optimizations and disable debug logs.

Reply:
this is part of the Sequelize template, i just cut off unnecessary parts of the config, the original looks like 
![image](https://user-images.githubusercontent.com/51543491/122803964-f6772a00-d2cf-11eb-8300-87dd50dad9b0.png)

i understand for what purpose development and productions modes used in general

Reply:
i updated the structure, to be more flexible with using .even file.